src/components/StorageOverview/TopConsumers/TopConsumers.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { LineChart, Row, Col } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../Dashboard/DashboardCard';
+import { StorageOverviewContext } from '../StorageOverviewContext/StorageOverviewContext';
+
+import { getTopConsumerVectorStats } from '../../../selectors';
+
+const TopConsumersBody = ({ columns }) => (
```

Comment:
It seems like a different component than `Dashboard/TopConsumers`. Can you rename it a bit? and maybe add it as a generic component to `Dashboard`?

Reply:
ok..i have just rebased